srv/modules/runners/push.py

```diff
@@ -211,6 +211,8 @@ def organize(self, filename):
         common = {}
         with open(filename, "r") as policy:
             for line in policy:
+                # strip comments from the end of the line
+                line = re.sub(' #.*$', '', line)
```

Comment:
right...nice catch. I'm wouldn't be worried about no space...that might be valid. But tabs or multiple spaces is def something that should be filtered. Patch coming up...